src/relay/analysis/get_calibration_data.cc

```diff
@@ -45,13 +45,14 @@ namespace relay {
 
 class Collector : public ExprRewriter {
  public:
-  explicit Collector(const IRModule& module) { glob_funcs_ = module->functions; }
+  explicit Collector(const IRModule& module) : module_(module) {}
 
   Expr Rewrite_(const CallNode* call, const Expr& post) final {
     // check if the function implementation is available
     // intrinsic functions are excluded for now
     if (call->op->IsInstance<GlobalVarNode>()) {
       auto var = Downcast<GlobalVar>(call->op);
+      auto glob_funcs_ = module_->functions;
```

Comment:
You actually don't need this.

module has `ContainGlobalVar` to check if var exists and `Lookup` to lookup for a var.